src/controllers/movies.js

```diff
@@ -0,0 +1,63 @@
+const { Prisma } = require("@prisma/client")
+const prisma = require('../utils/prisma')
+
+const getMovies = async (req, res) => {
+    const movies = await prisma.movie.findMany({
+      include: {
+        screenings: true
+      }
+    });
+    res.status(200).json({ movies })
+  }
+
+  const createMovie = async (req, res) => {
+    const createdMovie = await prisma.movie.create({
+        data:{
+            title: "Top Gun",
+            runtimeMins: 110
+        },
+        include: {
+            screenings: true
+          }
+    });
+    res.status(201).json({ movie: createdMovie });
+  }
```

Comment:
I would maybe try to make this createMovie with variables instead of using the test as a reference.

Reply:
Agree with Carolina you should make this with variables